packages/vue/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@neutrinojs/vue",
+  "version": "7.3.2",
+  "description": "Neutrino preset adding Vue.js support.",
+  "main": "src/index.js",
+  "author": "Capi Etheriel <barraponto@gmail.com> (https://barraponto.blog.br)",
+  "license": "GPL-3.0",
```

Comment:
In order for us to use the code as-is from the original repo, we need to re-license this new version as MPL-2.0. @barraponto do we have your permission for that?